### ISSUE TITLE
The pretrained weights download url is broken

### ISSUE THREAD

**Issue by xiemin0914** [inconst/faceswap#2]
hi inconst, the pretrained weights download url is broken, can you upload it again? thank you

**Comment by inconst** [comment]
Yeah, I can't seem to find the models in my google drive, something happened to them. I will notify you if I find them on my PC
